index.js

```diff
@@ -0,0 +1,6 @@
+/* jshint node: true */
+'use strict';
+
+module.exports = {
+  name: 'model-fragments'
```

Comment:
It should be ok. I've run it in my applications and I think @workmanw has tried it in his.